accord-project-technical-charter.md

```diff
@@ -10,92 +10,49 @@ a. The mission of the Project is to develop open source software tools for legal
 
 b. The scope of the Project includes collaborative development under the Project License (as defined herein) supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source project.
 
-## 2. Governing Board
 
-a. The Governing Board voting representatives shall consist of:
+## 2. Technical Steering Committee
 
-i. Two (2) directors of the Project; and 
-ii. The Chair of the the Technical Steering Committee.
+The Project shall be governed by the Technical Steering Committee (TSC) and the TSC shall be responsible for:
 
-b. Conduct of Meetings
+    (a. approving a budget directing the use of any funds raised by, or used by, AP;
 
-i. Governing Board meetings shall be limited to the Governing Board representatives and follow the requirements for quorum and voting established by the Governing Board.
+    b. electing a Chair of AP to preside over TSC meetings, authorize expenditures approved by the budget and manage any necessary day-to- day operations;
+     
+    c. electing a treasurer with authority for signing checks and responsiblity for credit card purchases;
```

Comment:
Can we generalise this? The main function will be approving expenses through the Linux Foundation. We don't hold a check book, or a credit card.

Reply:
If they hold the credit cards, check book we may not need a treasuurer - Why not just take it out?